libraries.json

```diff
@@ -579,6 +653,13 @@
     "description": "An alternative side effect model for Redux apps.",
     "language": "JavaScript"
   },
+  "Rex": {
+    "imports": ["Rex"],
+    "technologies": ["Automation"],
```

Comment:
Can DevOps be added?

Reply:
Yes, I guess that label would help to orient people.